custom/ewsghana/api.py

```diff
@@ -689,6 +750,15 @@ def sms_user_sync(self, ews_smsuser, **kwargs):
         if ews_smsuser.role:
             sms_user.user_data['role'] = [ews_smsuser.role]
 
+        if ews_smsuser.phone_numbers:
+            sms_user.user_data['connections'] = []
+            for connection in ews_smsuser.phone_numbers:
+                sms_user.user_data['connections'].append({
```

Comment:
Moreover if we are setting during migration test backend to user with test backend in v1 and other users use default backend, I'm not quite sure that we need this information saved in user.